### DISCUSSION TITLE
Push Button with two relays

### DISCUSSION THREAD

**Discussion by LeonvG** [arendst/Tasmota#11278] [Q&A] [answered]
Hi!

I have one button and two relays.
How I can implement sequential switching. Like the following scenario:

Initial state:
Relay1 - OFF, Relay1 - OFF

1st Button push - Relay1 - ON, Relay2 - OFF
2nd Button push - Relay1 - OFF, Relay2 - ON
3rd Button push - Relay1 - ON, Relay2 - OFF
4th Button push - Relay1 - OFF, Relay2 - ON

Button HOLD - Relay1 - OFF, Relay2 - OFF

**Comment by ascillato2** [comment, marked as answer]
Hi,

Yes it is possible to implemented that behavior using rules. You can use a VAR for remembering those states and with rules you can check that behavior. Please, check the docs on how to use rules. Thanks.

**Reply by LeonvG** [reply]
OK thank you very much. Gonna try it soon

**Comment by guydvir2** [comment]
@LeonvG what you ask, is already answered
select your state in as shown:
a) https://tasmota.github.io/docs/Blinds-and-Shutters/
and then select your button functionality here:
b) https://tasmota.github.io/docs/Blinds-and-Shutters/#button-control
